src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Baselines.cs

```diff
@@ -965,5 +965,153 @@ public abstract class AbstractType_CannotInit
+                    public List<Dictionary<string, string[,]>> SkipProp27 { get; set; }
+
+                    public Func<string>[] SkipProp28 { get; set; }
+                    public Action<string>[] SkipProp29 { get; set; }
+                    public myMethodDelegate[] SkipProp30 { get; set; }
+                    public MethodInfo[] SkipProp31 { get; set; }
+                    public ConstructorInfo[] SkipProp32 { get; set; }
+                    public SerializationInfo[] SkipProp33 { get; set; }
+                    public IntPtr[] SkipProp34 { get; set; }
+                    public UIntPtr[] SkipProp35 { get; set; }
+
+                    //
+                    // Repeat the above properties, but with a nullability annotation.
+                    //
+
+                    public Action<string>? SkipProp36 { get; set; }
+                    public List<Action<string>?> SkipProp37 { get; set; }
+                    public List<Dictionary<string, Action<string>?>> SkipProp38 { get; set; }
+
+                    public Func<int>? SkipProp39 { get; set; }
+                    public List<Func<int>?> SkipProp40 { get; set; }
+                    public List<Dictionary<string, Func<string>?>> SkipProp41 { get; set; }
+
+                    public myMethodDelegate? SkipProp42 { get; set; }
+                    public List<myMethodDelegate?> SkipProp43 { get; set; }
+                    public List<Dictionary<string, myMethodDelegate?>> SkipProp44 { get; set; }
+
+                    public IntPtr? SkipProp45 { get; set; }
+                    public List<IntPtr?> SkipProp46 { get; set; }
+                    public List<Dictionary<string, IntPtr?>> SkipProp47 { get; set; }
+
+                    public UIntPtr? SkipProp48 { get; set; }
+                    public List<UIntPtr?> SkipProp49 { get; set; }
+                    public List<Dictionary<string, UIntPtr?>> SkipProp50 { get; set; }
+
+                    public SerializationInfo? SkipProp51 { get; set; }
+                    public List<SerializationInfo?> SkipProp52 { get; set; }
+                    public List<Dictionary<string, SerializationInfo?>> SkipProp53 { get; set; }
+
+                    public MethodInfo? SkipProp54 { get; set; }
+                    public List<MethodInfo?> SkipProp55 { get; set; }
+                    public List<Dictionary<string, MethodInfo?>> SkipProp56 { get; set; }
+
+                    public ConstructorInfo? SkipProp57 { get; set; }
+                    public List<ConstructorInfo?> SkipProp58 { get; set; }
+                    public List<Dictionary<string, ConstructorInfo?>> SkipProp59 { get; set; }
+
+                    public string[,]? SkipProp60 { get; set; }
+                    public List<string[,]?> SkipProp61 { get; set; }
+                    public List<Dictionary<string, string[,]?>> SkipProp62 { get; set; }
+
+                    public Func<string>?[] SkipProp63 { get; set; }
```

Comment:
Similarly, you could try reducing the skipped property to just one and fill in using the input from the theory.

Reply:
The drawback of this will make the test much slower as we compare the result against the baseline.

Reply:
I can pass the whole list as one shot through to the theory.